service/middleware.go

```diff
@@ -393,6 +423,22 @@ func (uis *UIServer) LoadProjectContext(rw http.ResponseWriter, r *http.Request)
 		})
 	}
 
+	grip.Debug(message.Fields{
+		"ticket":      "EVG-14261",
+		"id":          id,
+		"step":        "LoadContext",
+		"func":        "LoadProjectContext",
+		"duration_ns": time.Since(start).Nanoseconds(),
```

Comment:
The other PR had duration as well as duration_ns, if that was useful and you want to have both

Reply:
nope, they're both the same so I kept 1